sedai/README.md

```diff
@@ -0,0 +1,47 @@
+**[Sedai][1] — Find out why production is the new playground with a free 14 day trial**
+### Continuous Autonomous Availability Management (CAAM)
+
+*Intelligent enough to detect, independent enough to act — that's the power of autonomous*
+
+## Overview
+
+As a Sedai customer you can integrate directly with DataDog to see selected notifications in your DataDog Dashboard.
+
+Sedai proactively manages your cloud environments for you through autonomous actions that prevent availability issues as well as optimize your resources’ performance — all in production. Our agentless and configuration-free platform infers your cloud environments and intelligently learns and adapts based on observed performance. From preventing downtime and managing SLOs to analyzing code releases and lowering cloud costs, learn how our autonomous platform can become a staple in your SRE tool kit.
+
+**Unlock the full potential of your microservices:**
+* Achieve 99.99% or better application availability
+* Autonomously reduce resource latency by 75%
+* Reduce cloud costs by 30% through autonomous optimization
+* Increase team productivity and agility by 3x
+
+### Core Features
+* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
+* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
+* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
+* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
+* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.
+
+Visit **[Sedai Inc.][1]** to learn more.
+
+### Technical Documentation
+Please review the [Sedai User Guide][4].
+## Setup
+Existing Sedai customers can receive notifications in DataDog about Sedai activity. Learn how to connect DataDog to Sedai in **[our documentation][6].**
```

Comment:
[6] links to https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications, not https://sedai.gitbook.io/sedai/sedai-user-guide/controls/integrations/notifications-integrations. There are two sections titled "Notifications" so I want to confirm we're linking to the right one!  

```suggestion

After connecting your cloud provider, monitoring provider, and notification provider accounts, Sendai identifies your topology and metrics and starts to analyze performance.

### Installation

Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai documentation][6].
```

Reply:
The two links are on purpose, our UI has 2 related settings pages that are involved in configuration of notifications.